cms/articles/templates/wagtailadmin/pages/edit.html

```diff
@@ -0,0 +1,8 @@
+{% extends "wagtailadmin/pages/edit.html" %}
+{% block extra_js %}
+    {{ block.super }}
+    {# We need to pass this data to our JavaScript, used in headline-figures-item-block.js #}
+    {% if page.figures_used_by_ancestor %}
+        {{ page.figures_used_by_ancestor|json_script:"figures-used-by-ancestor" }}
+    {% endif %}
+{% endblock %}
```

Comment:
question: is there any way you can do this via the admin form or `insert_editor_js`?

My worry here is that this could be brittle especially if we need to override the edit template somewhere else.

Reply:
I couldn't find a way to do it using `insert_editor_js` because I need access to the current page instance.